Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -571,20 +582,28 @@ open class MapView: UIView {
 
     @_spi(Metrics) public var metricsReporter: MapViewMetricsReporter?
     private func updateFromDisplayLink(displayLink: CADisplayLink) {
+        let trace = OSLog.platform.beginInterval(name: "MapView.displayLink")
+        defer { trace.end() }
+
         metricsReporter?.beforeDisplayLinkCallback(displayLink: displayLink)
```

Comment:
We already have a metricsReporter protocol in here. I was thinking that we might adopt it for signpost needs.
Right now it is used only in benchmarking to calculate fps and drawing time.
I like an idea of optional variable here as it would reduce performance impact as much as possible in case of disabled metrics (which should be default for SDK)